terra/reports.py

```diff
@@ -68,7 +81,15 @@ def merge_data(rows, data):
     for subunit in data["subunits"].values():
         for employee in subunit["employees"].values():
             try:
-                employee.data = rows.get(employee_id=employee.id)
+                employee.data = rows.get(id=employee.id)
```

Comment:
@joshuago78 At present the build fails.  First time was "MySQL not available" which was very odd given that nothing relevant changed, so I just re-ran it in Travis.  Now several tests fail with "Subquery returns more than one row".

Several errors mention this line, for what that's worth.